hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/LogRoller.java

```diff
@@ -27,8 +27,8 @@
 import java.util.Map.Entry;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
+import org.apache.hadoop.hbase.HBaseIOException;
```

Comment:
Look like this is not used, and make a checkstyle warning

Reply:
Thank you @binlijin